lib/coverage.js

```diff
@@ -0,0 +1,108 @@
+const config = require('../lib/config')
```

Comment:
This file `coverage.js` is a copy of `build.js` and is not strictly necessary, it just made sure I didn't break anything else whilst poking around.

The only noteworthy change compared to `build.js` is the `utils.runCoverage` step instead of `utils.buildTarget`

Reply:
missed this comment the first time, but anyway the comments should hopefully help you clean it up

Reply:
@bridiver cheers for the feedback :) So my thinking was building with `npm run test -- brave_unit_tests` should provide optional flags/args to also generate coverage profile data, provided it doesn't impact/cause issues elsewhere.  WDYT?